data-serving/scripts/converters.py

```diff
@@ -41,32 +41,27 @@ def convert_range(
         range_min = parse_fn(value_range[0])
         if len(value_range) == 1:
             return {
-                'range': {
-                    'max': format_fn(range_min)
-                }
+                'start': format_fn(range_min)
```

Comment:
Isn't max == end, not start?

Associated nit: slightly confusing to call this local value `range_min`.

Reply:
what is wrong with me

also renamed the local vars